### DISCUSSION TITLE
Delay only exit animation

### DISCUSSION THREAD

**Discussion by xzilja** [nandorojo/moti#161] [Q&A] [answered]
First of all, thank you for this amazing library, having something like framer motion, but built for react-native gives so much freedom for creativity!

I wanted to ask if there is a way to only apply delay on exit animation (i.e. for component that has from -> animate and exit) without introducing custom logic and state variable to track this?

**Comment by nandorojo** [comment, marked as answer]
yes, use the `exitTransition` prop
